### ISSUE TITLE
Add Batch Normalization

### ISSUE THREAD

**Issue by cjacoby** [ejhumphrey/harmonic-cnn#22]
See http://lasagne.readthedocs.io/en/latest/modules/layers/normalization.html#lasagne.layers.BatchNormLayer 
and this helper function: http://lasagne.readthedocs.io/en/latest/modules/layers/normalization.html#lasagne.layers.batch_norm
